compact.go

```diff
@@ -217,7 +217,7 @@ func (c *LeveledCompactor) plan(dms []dirMeta) ([]string, error) {
 	for i := len(dms) - 1; i >= 0; i-- {
 		meta := dms[i].meta
 		if meta.MaxTime-meta.MinTime < c.ranges[len(c.ranges)/2] {
-			break
+			continue
```

Comment:
There's no need to change this, as blocks get smaller as you approach now.

Reply:
But what if the distance between the first block and the later blocks is large enough that the previous compactions never cover the first block(so some later blocks may be larger than the first one)?

Reply:
Your proposed change wouldn't affect that.

Reply:
@naivewong 	Brian is right about that one. This is sort of a limiter to avoid running compactions too often.